src/grt/grt-wave_opt_file-parse.adb

```diff
@@ -59,28 +60,28 @@ package body Grt.Wave_Opt_File.Parse is
                         return Boolean;
 
    -- Parse the line where the version is set
-   procedure Parse_Version (Line : String_Access);
+   procedure Parse_Version (Line : String; Line_Pos : Positive);
```

Comment:
Actually, I first named it Lineno, but after reaching one point, I found 
out that I need the column number too, but with this naming convention, 
it results to Columnno which looked weird to me. Maybe that was just a 
matter of view and Columnno is fine. That's why I changed Lineno to 
Line_Pos. Maybe renaming them to LineNo, ColumnNo makes that look better ?

Le 27. 07. 16 à 19:25, tgingold a écrit :
>
> In src/grt/grt-wave_opt_file-parse.adb 
> <https://github.com/tgingold/ghdl/pull/124#discussion_r72483682>:
>
> > @@ -59,28 +60,28 @@ package body Grt.Wave_Opt_File.Parse is
> >                          return Boolean;
> >
> >     -- Parse the line where the version is set
> > -   procedure Parse_Version (Line : String_Access);
> > +   procedure Parse_Version (Line : String; Line_Pos : Positive);
>
> Line_Pos sounds to be a position within Line. For line number, Lineno 
> is more common. (Not a blocking comment),
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/tgingold/ghdl/pull/124/files/eae288207d481e422e395aa7d310fa629d20432b#r72483682>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ATNePsxEX8SMQNaYSjb54lOcASQ0p8l1ks5qZ5ShgaJpZM4JVgTD>.
>